web/src/settings_invites.ts

```diff
@@ -225,8 +225,22 @@ function do_resend_invite({$row, invite_id}: {$row: JQuery; invite_id: string}):
             dialog_widget.hide_dialog_spinner();
             dialog_widget.close();
             $resend_button.prop("disabled", true);
-            $resend_button.text($t({defaultMessage: "Sent!"}));
-            $resend_button.removeClass("resend button-warning").addClass("sea-green");
+            $resend_button
+                .removeClass("resend icon-button-neutral")
+                .addClass("check icon-button-success");
+            $resend_button.find("i").removeClass("zulip-icon-send-dm").addClass("zulip-icon-check");
```

Comment:
Any reason to not have both icons present and hide and show it accordingly? Seems easy to understand and will help in avoiding future bugs.